### DISCUSSION TITLE
Vote skip intro in multiplayer

### DISCUSSION THREAD

**Discussion by Sclafus** [ppy/osu#26273] [Ideas] [answered]
in #14260 it was discussed the possibility of "skipping" in multiplayer lobbies, which resulted in the current implementation 
![image](https://github.com/ppy/osu/assets/31241607/1be6e100-6b49-40d9-9291-69529fdab613)
This is a lobby-wide option, some less-aware hosts may not see this option until someone puts a map with a 30 minute intro like some difficulties of [The Globglogabgalab](https://osu.ppy.sh/beatmapsets/753730#osu/1597642) 
I feel like most players would like a stable-like (or even solo-like) skip button, which skips only if everyone voted to skip the intro.

**Comment by marioman63** [comment]
I could have sworn there was a small period of time just before this option existed where vote skip was added, but then mysteriously removed later on.

**Reply by bdach** [reply]
There was not. At least not on lazer.

**Comment by ziv-vy** [comment]
Bump due to matchmaking update and #35356.

**Comment by Gooeys1** [comment, marked as answer]
#35542
